gnocchi/opts.py

```diff
@@ -59,6 +59,49 @@ def __getitem__(self, key):
 
 def list_opts():
     return [
+        ("DEFAULT", (
+            cfg.BoolOpt(
+                'debug',
+                short='d',
+                default=False,
+                help='If set to true, the logging level will be set to '
+                'DEBUG.'),
+            cfg.BoolOpt(
+                'verbose',
+                short='v',
+                default=False,
+                deprecated_for_removal=True,
```

Comment:
Hm it's deprecated in oslo.log so I think it's why I pasted it. But the option is not actually used, I need to fix that.